### ISSUE TITLE
Allow file sync between local folders? - including ext SD card access.

### ISSUE THREAD

**Issue by bushrang3r** [TheAkki/Synctool#29]

**Comment by theScrabi** [comment]
Accessing SD card only works by using the [Android Storage Framework](https://developer.android.com/guide/topics/providers/document-provider) this however is not yet supported by the [ReadRemoteFilesOperation](https://github.com/TheAkki/Synctool/blob/master/app/src/main/java/theakki/synctool/OwnCloud/ReadRemoteFilesOperation.java).

**Comment by TheAkki** [comment]
Yes, it's possible to sync files between 2 local folders. It's not necessary to use ReadRemoteFilesOperations.

**Comment by bushrang3r** [comment]
Thanks for the quick response. 
So, do you mean that syncing between internal memory and ext SD card can't be done?

**Comment by TheAkki** [comment]
I think it can be done (Theoretically) and should be done at the end. 
But i must check which is my start folder for browse.
Also i have to check whats happen when you select a folder as string in GUI. May be it works with Work-a-rounds.
Also i have to check whats happen when using smartphone without root.
